client/jetpack-onboarding/steps/stats.jsx

```diff
@@ -96,13 +97,10 @@ class JetpackOnboardingStatsStep extends React.Component {
 		const connectUrl = addQueryArgs(
 			{
 				from: 'jpo',
-				redirect_after_auth: [ basePath, stepName, siteSlug ].join( '/' ),
-				url: siteUrl,
-				// TODO: add a parameter to the JPC to redirect back to this step after completion
-				// and in the redirect URL include the ?action=activate_stats parameter
-				// to actually trigger the stats activation action after getting back to JPO
+				redirect_after_auth: addQueryArgs( { action: 'activate_stats' }, window.location.href ),
+				calypso_env: config( 'env_id' ),
 			},
-			'/jetpack/connect'
+			siteUrl + '/wp-admin/admin.php?page=jetpack&connect_url_redirect=true'
```

Comment:
I know we're trying to avoid accessing the const from `jetpack-connect` here, but it's kind of a shame to use a string literal (without even commenting) when this is available as a const somewhere. I'd be actually more in favor of importing and using that const here, and adding a `TODO` to move the `constants` file to a more generic directory.

Reply:
Sure! Let's just get it working first 😆